src/components/MenuBar/Menu/Menu.test.tsx

```diff
@@ -50,17 +58,60 @@ describe('the Menu component', () => {
       expect(wrapper.find(ExpandMoreIcon).exists()).toBe(true);
       expect(wrapper.find(MoreIcon).exists()).toBe(false);
     });
+
+    it('should not render the Flip Camera button', () => {
+      const wrapper = shallow(<Menu />);
+      expect(wrapper.find(FlipCameraIcon).exists()).toBe(false);
+    });
   });
 
   describe('on mobile devices', () => {
     beforeAll(() => {
       mockUseMediaQuery.mockImplementation(() => true);
+      mockUseFlipCameraToggle.mockImplementation(() => ({
+        flipCameraDisabled: false,
+        flipCameraSupported: true,
+      }));
     });
 
     it('should render the correct icon', () => {
       const wrapper = shallow(<Menu />);
       expect(wrapper.find(ExpandMoreIcon).exists()).toBe(false);
       expect(wrapper.find(MoreIcon).exists()).toBe(true);
     });
+
+    it('should render non-disabled Flip Camera button when facingMode is supported', () => {
+      const wrapper = shallow(<Menu />);
+      expect(wrapper.find(FlipCameraIcon).exists()).toBe(true);
+      expect(
+        wrapper
+          .find(MenuItem)
+          .at(0)
+          .prop('disabled')
+      ).toBe(false);
+    });
+
+    it('should render a disabled Flip Camera button when facingMode is supported, but videoTrack is undefined', () => {
```

Comment:
Nit: I think the text in the `it(...)` functions can be updated here. The Menu component isn't aware of what `facingMode` or `videoTrack` is. The Menu component only sees `flipCameraDisabled` and `flipCameraSupported`, so I think those are the variables that should be mentioned in the `it(...)` functions (on lines 83 and 109 too).